build/gen_stub.php

```diff
@@ -1480,23 +1483,380 @@ private function getFlagsAsArginfoString(): string
+        $refnamediv->appendChild(new DOMText("\n  "));
+        $refname = $doc->createElement('refname', $this->name->__toString());
+        $refnamediv->appendChild($refname);
+        $refnamediv->appendChild(new DOMText("\n  "));
+        $refpurpose = $doc->createElement('refpurpose', 'Description');
+        $refnamediv->appendChild($refpurpose);
+
+        $refnamediv->appendChild(new DOMText("\n "));
+        $refentry->append($refnamediv, $REFSEC1_SEPERATOR);
+
+        /* Creation of <refsect1 role="description"> */
+        $descriptionRefSec = $this->generateRefSect1($doc, 'description');
+
         $methodSynopsis = $this->getMethodSynopsisElement($funcMap, $aliasMap, $doc);
         if (!$methodSynopsis) {
             return null;
         }
+        $descriptionRefSec->appendChild($methodSynopsis);
+        $descriptionRefSec->appendChild(new DOMText("\n  "));
+        $undocumentedEntity = $doc->createEntityReference('warn.undocumented.func');
+        $descriptionRefSec->appendChild($undocumentedEntity);
+        $descriptionRefSec->appendChild(new DOMText("\n  "));
+        $returnDescriptionPara = $doc->createElement('para');
+        $returnDescriptionPara->appendChild(new DOMText("\n   Description\n  "));
+        $descriptionRefSec->appendChild($returnDescriptionPara);
+
+        $descriptionRefSec->appendChild(new DOMText("\n "));
+        $refentry->append($descriptionRefSec, $REFSEC1_SEPERATOR);
+
+        /* Creation of <refsect1 role="parameters"> */
+        $parametersRefSec = $this->getParameterSection($doc);
+        $refentry->append($parametersRefSec, $REFSEC1_SEPERATOR);
+
+        /* Creation of <refsect1 role="returnvalues"> */
+        if (!$this->name->isConstructor() && !$this->name->isDestructor()) {
+            $returnRefSec = $this->getReturnValueSection($doc);
+            $refentry->append($returnRefSec, $REFSEC1_SEPERATOR);
+        }
+
+        /* Creation of <refsect1 role="errors"> */
+        $errorsRefSec = $this->generateRefSect1($doc, 'errors');
+        $errorsDescriptionPara = $doc->createElement('para');
+        $errorsDescriptionPara->appendChild(new DOMText("\n   When does this function issue E_* level errors, and/or throw exceptions.\n  "));
+        $errorsRefSec->appendChild($errorsDescriptionPara);
+        $errorsRefSec->appendChild(new DOMText("\n "));
+
+        $refentry->append($errorsRefSec, $REFSEC1_SEPERATOR);
+
+        /* Creation of <refsect1 role="changelog"> */
+        $changelogRefSec = $this->getChangelogSection($doc);
+        $refentry->append($changelogRefSec, $REFSEC1_SEPERATOR);
+
+        $exampleRefSec = $this->getExampleSection($doc);
+        $refentry->append($exampleRefSec, $REFSEC1_SEPERATOR);
+
+        // TODO Notes section?
+
+        /* Creation of <refsect1 role="seealso"> */
+        $seeAlsoRefSec = $this->generateRefSect1($doc, 'seealso');
+
+        /* TODO Actually generate a markup for class names, functions and links?
```

Comment:
Maybe..., If the seealso section is needed, then it's easier if its content also has a template. When the section is unnecessary, then people will remove it altogether anyway.

Reply:
It's a good idea, but I was trying to get this out the door ASAP